lib/trento/discovery/payloads/host_discovery_payload.ex

```diff
@@ -29,6 +29,7 @@ defmodule Trento.Discovery.Payloads.HostDiscoveryPayload do
     field :fully_qualified_domain_name, :string
     field :prometheus_targets, :map
     field :systemd_units, {:array, :map}, default: []
+    field :last_boot_timestamp, :utc_datetime, default: nil
```

Comment:
naming:
every other timestamp field is named finishing with the `_at` suffix, which would become this to `last_boot_at` or `last_boot_timestamp_at`.
What do you think?

Reply:
To be honest, I copied from this field in the host model

https://github.com/trento-project/web/blob/c9d373694e922b3b81345af89523e87b6d3331a5/lib/trento/hosts/projections/host_read_model.ex#L70

and in the openapi schema as well

https://github.com/trento-project/web/blob/0aae0ae793182928fec0f455a2707a81b6df4350/lib/trento_web/openapi/v1/schema/host.ex#L177